### DISCUSSION TITLE
Предложение

### DISCUSSION THREAD

**Discussion by SlalomJohn** [OleksiyPenkov/MyHomeLib#25] [Ideas]
0. Иконка бинарника сейчас штатная от Delphi как я понял
1. Сделать возможность отслеживания (избранное) для серий и авторов
2. Включить возможность самостоятельной авторизации на либрусек/флибусты (у кого есть акки там)

**Comment by OleksiyPenkov** [comment]
0 - это чтобы от релиза отличать )))
1 - нужно менять структуру БД, а это табу. По крайней мере, пока. Сюда же вложенные серии, типы серий и т.п.
2 - это не понял.

**Comment by SlalomJohn** [comment]
> 0 - это чтобы от релиза отличать ))) 1 - нужно менять структуру БД, а это табу. По крайней мере, пока. Сюда же вложенные серии, типы серий и т.п. 2 - это не понял.

Ну по п.2 - если книги нет (или архива) ранее MHL загружал книги с сайта (librusec/flibusta), но как я понял - с встроенным аккаунтом или без оного. Так вот - сделать бы возможность внесения логина и пароля от сайтов - для онлайн загрузки - ы?

По п.1 - как я понимаю - структуру внутренней БД менять надо? А что мешает-то? Потому как избранная книга - вещь более чем бесполезная. А вот избранный автор или серия - весьма и зело полезно и нужно...

**Comment by OleksiyPenkov** [comment]
Я бы посмотрел, чего у них там нынче с авторизацией, да пароль свой от либрусека потерял, а восстановление у них не работает )))
Насчёт изменения БД, мешает 1) потеря совместимости 2) объем работы. Я пока не готов.

**Comment by OleksiyPenkov** [comment]
Но я подумаю

**Comment by mexx1111** [comment]
Прошу рассмотреть предложение о выдаче авторам собственных оценок, как книгам, и возможности использовать эти оценки в поисковых формах и отборе в списке авторов

**Comment by SlalomJohn** [comment]
> Прошу рассмотреть предложение о выдаче авторам собственных оценок, как книгам, и возможности использовать эти оценки в поисковых формах и отборе в списке авторов

А для чего? Все равно как я понимаю, эти оценки остаются только локально у вас на ПК и в онлайн не уходят.

**Comment by mexx1111** [comment]
> > > Прошу рассмотреть предложение о выдаче авторам собственных оценок, как книгам, и возможности использовать эти оценки в поисковых формах и отборе в списке авторов
> > А для чего? Все равно как я понимаю, эти оценки остаются только локально у вас на ПК и в онлайн не уходят.
> Естественно это локальные мои оценки и для меня.
При чём тут вообще онлайн и зачем он??? Это же моя библиотека.
Просто я, например, был бы не против, подыскивая новые книги для отправки на устройство (да хоть бы даже для скачивания, в случае онлайн-коллекции) отобрать только авторов, к примеру без оценки или с оценкой >=3

**Comment by montegry** [comment]
Очень извиняюсь, а как это устанавливать? Попытка собрать установщик падает из-за путей файлов. Я что-то делаю не так...

**Reply by OleksiyPenkov** [reply]
Зачем собирать установщик? Все работает и в портативном варианте.

**Reply by montegry** [reply]
Вы меня простите, я падал головой, глуповат. А как запускать программу? С гита я ее клонировал, а дальше?

**Reply by OleksiyPenkov** [reply]
Готовые установщики и портабельные сборки https://github.com/OleksiyPenkov/MyHomeLib/releases/tag/2.3.5

**Reply by OleksiyPenkov** [reply]
Если хочется собрать все самому, вот примерная инструкция
https://github.com/OleksiyPenkov/MyHomeLib/discussions/30?converting=1

**Reply by montegry** [reply]
Спасибо!
